WF_Proton_Launcher/Hashlist.cpp

```diff
@@ -94,7 +94,7 @@ void Hashlist::hashLocal()
 		if (mLocalHashes.count(filename) > 0)
 			continue;
 
-		if (filename.substr(0, 15) == "/Cache.Windows/")
+		if (filename.substr(0, 15) == "/Cache.Windows/" || filename.substr(0, 13) == "/Launcher.exe")
```

Comment:
That change will not do exactly what you think, if you look in `getDiffering()` you'll see that it will already explicitly ignore Launcher.exe regardless of what the hash values say.

Though skipping hashing the launcher executable is probably not a stupid idea, so I'm okay with keeping it here.